extensions/amp-iframe/0.1/test/test-amp-iframe.js

```diff
@@ -550,4 +553,16 @@ describe('amp-iframe', () => {
     expect(isAdLike(e(335, 100))).to.be.true;
     expect(isAdLike(e(341, 100))).to.be.false;
   });
+
+  it('should not render fixed ad', () => {
+    return getAmpIframe({
+      src: iframeSrc,
+      sandbox: 'allow-scripts allow-same-origin',
+      width: 300,
+      height: 250,
+      position: 'fixed',
+    }).then(amp => {
+      expect(amp.iframe).to.be.null;
+    }).catch(() => {});
```

Comment:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-iframe/0.1/test/test-amp-iframe.js#L102
The test call `pollForLayout` when creating the amp-iframe, it looks for an element with class `-amp-layout` but there's none. I don't actually understand the amp-iframe test: when do we call `layoutCallback` of amp-iframe in the test?